common/src/calculate-cpmm.ts

```diff
@@ -577,14 +576,67 @@ export function addCpmmMultiLiquidityAnswersSumToOne(
   return newPools
 }
 
+// 0 to 1. The minimum fraction of the pool that we let remove. Extreme ratios with extreme p values have bad liquidity i think.
+export const FRACTION_OF_POOL_MIN = 0.1
+// Must be at least this many yes and no shares
+export const MINIMUM_LIQUIDITY = 100
+
+export function removeCpmmLiquidity(
+  pool: { [outcome: string]: number },
+  p: number,
+  amount: number
+) {
+  const { newPool, liquidity, newP } = addCpmmLiquidity(pool, p, -1 * amount)
+
+  let error = ''
+
+  if (newPool.YES < MINIMUM_LIQUIDITY || newPool.NO < MINIMUM_LIQUIDITY) {
+    error = 'below minimum'
```

Comment:
we don't really use this style of error handling (versus throwing an exception) anywhere else on the backend to my knowledge...

Reply:
this function is used on the frontend and I want it to still return what the pool values would be even if it is below 100